scripts/deploy-keccak256/deploy.js

```diff
@@ -165,13 +183,40 @@ async function main() {
     znsResolverProxy: event[4],
     zkbnbProxy: event[5],
     upgradeGateKeeper: event[6],
-    BUSDToken: BUSDToken.address,
-    LEGToken: LEGToken.address,
-    REYToken: REYToken.address,
+    additionalZkBNB: event[7],
     ERC721: ERC721.address,
     znsPriceOracle: priceOracle.address,
     DefaultNftFactory: DefaultNftFactory.address,
     upgradeableMaster: upgradeableMaster.address,
+    utils: contractFactories.Utils.address,
+  });
+
+  // Save contract constructor arguments to JSON for verify
+  saveConstructorArgumentsForVerify('info/constructor.json', {
+    proxy: [event[0], [abi.encode(['address'], [deployFactory.address])]],
```

Comment:
deployFactory is **not** a Proxy contract. It deploys several proxy contracts inside though. This is probably incorrect

See https://testnet.bscscan.com/address/0xbfdf8bcf5a0ced47e305f07b7871094ef995bfcc#code. This is a proxy contract but it targets Governance.sol. If you look closely, it has all the contracts verified and it is not Governance.sol

Reply:
<img width="366" alt="image" src="https://user-images.githubusercontent.com/117076605/215067133-0340fe17-e795-420e-9d33-86a70315541b.png">
Here is to verify the Proxy contract, after verification you need to manually click this action

Governance Contract will be performed in the second verification